harness/determined/pytorch/_lightning/adapter.py

```diff
@@ -0,0 +1,184 @@
+import inspect
+from typing import Any, Dict, List, Sequence, Tuple, Union, cast
+
+import pytorch_lightning as pl
+import torch
+from pytorch_lightning.trainer.optimizers import TrainerOptimizersMixin
+from pytorch_lightning.utilities.model_utils import is_overridden
+from torch.optim.lr_scheduler import _LRScheduler
+from torch.optim.optimizer import Optimizer
+
+from determined.errors import UnsupportedException
+from determined.monkey_patch import has_param, monkey_patch
+from determined.pytorch import LRScheduler, PyTorchCallback, PyTorchTrial, PyTorchTrialContext
+from determined_common import check
+
+TorchData = Union[Dict[str, torch.Tensor], Sequence[torch.Tensor], torch.Tensor]
+
+
+def check_compat(lm: pl.LightningModule) -> None:
+    prefix = "Unsupported usage in PLAdapter: "
```

Comment:
```suggestion
    prefix = "Unsupported usage in PTLAdapter: "
```

Reply:
hmm  I changed these when we talked about using PL instead of PTL everywhere